drivers/entropy/entropy_cc13xx_cc26xx.c

```diff
@@ -26,6 +31,13 @@ struct entropy_cc13xx_cc26xx_data {
 	struct k_sem sync;
 	struct ring_buf pool;
 	uint8_t data[CONFIG_ENTROPY_CC13XX_CC26XX_POOL_SIZE];
+#ifdef CONFIG_SYS_POWER_MANAGEMENT
+	Power_NotifyObj postNotify;
```

Comment:
Is the code style different here? I don't normally see camel case

Reply:
Yes, this is the coding style used in the TI HAL, more specifically in the TI SimpleLink SDK where this type is defined. I can make the variable name non-camel-case however. Maybe that is what you are referring to - let me fix that.